src/gpu/intel/sycl/l0/utils.cpp

```diff
@@ -394,7 +394,18 @@ status_t init_gpu_hw_info(impl::engine_t *engine, ze_device_handle_t device,
     stepping_id = product.stepping;
 
     mayiuse_systolic = false;
-    CHECK(get_l0_device_enabled_systolic_intel(device, mayiuse_systolic));
+    if (get_l0_device_enabled_systolic_intel(device, mayiuse_systolic)
+            != status::success)
+        mayiuse_systolic = false;
+
+    /* Some old drivers do not report systolic availability. Manually override
```

Comment:
Please add some version information to the comment, so that it is easier to determine when this can be removed in the future.

Reply:
There's actually already a comment in `get_l0_device_enabled_systolic_intel` about that very thing.